_includes/live-demos/comments-callback/index.html

```diff
@@ -0,0 +1,33 @@
+<textarea id="callback-mode">
+    <h2>Welcome to Tiny Comments!</h2>
+    <p>Please try out this demo of our Tiny Comments premium plugin.</p>
+    <ol>
+      <li>Highlight the content you want to comment on.</li>
+      <li>Click the <em>Add Comment</em> icon in the toolbar.</li>
+      <li>Type your comment into the text field at the bottom of the Comment sidebar.</li>
+      <li>Click <strong>Save</strong>.</li>
+    </ol>
+    <p>Your comment is then attached to the text, exactly like this!</p>
+    <p>If you want to take Tiny Comments for a test drive in your own environment, Tiny Comments is one of the premium plugins you can try for free for 30 days by signing up for a Tiny account. Make sure to check out our documentation as well.</p>
+    <p>And if you are ready to buy, you can either purchase Tiny Comments a la carte or as part of your Tiny Pro subscription. If you are a current cloud user, Tiny Comments has already been added to your key, and if you are a self-hosted user, Tiny Comments is now available in the latest download which you can access in My Account.</p>
```

Comment:
We should remove this line, which appears to be a legacy plan.
"And if you are ready to buy, you can either purchase Tiny Comments a la carte or as part of your Tiny Pro subscription"

Reply:
```suggestion
```